File: be-online-shop/src/main/java/com/mfauzirh/beonlineshop/dto/OrderResponse.java
package com.mfauzirh.beonlineshop.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.UUID;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class OrderResponse {
    private Long orderId;
    private UUID orderCode;
    private LocalDateTime orderDate;
    private Long totalPrice;
    private Integer quantity;
    private Long customerId;
    private String customerName;
    private String customerAddress;
    private String customerPic;
    private String customerPhone;
    private Long itemId;
    private String itemName;
    private Integer stock;
    private Long price;
}

File: fe-online-shop/src/app/models/order-response.ts
export interface OrderResponse {
  orderId: number;
  orderCode: string
  orderDate: string;
  totalPrice: number;
  quantity: number;
  customerId: number;
  customerName: string;
  customerAddress: string;
  customerPic: string;
  customerPhone: string;
  itemId: number;
  itemName: string;
  stock: number;
  price: number;
}